### DISCUSSION TITLE
interface eth1 prevented this program to run

### DISCUSSION THREAD

**Discussion by seb4nihel** [sakai135/wsl-vpnkit#111] [Q&A] [answered]
Hello when I try to use wsl-vpnkit I always get the same behaviour:

starting wsl-vpnkit
cleaning up iptables...
iptables v1.8.7 (legacy): can't initialize iptables table `nat': Table does not exist (do you need to insmod?)
Perhaps iptables or your kernel needs to be upgraded.
iptables v1.8.7 (legacy): can't initialize iptables table `nat': Table does not exist (do you need to insmod?)
Perhaps iptables or your kernel needs to be upgraded.
iptables v1.8.7 (legacy): can't initialize iptables table `nat': Table does not exist (do you need to insmod?)
Perhaps iptables or your kernel needs to be upgraded.
iptables cleanup done
restoring WSL 2 ip route...
ip: RTNETLINK answers: Permission denied
ip: either "to" is duplicate, or "10.58.194.11" is garbage
restored WSL 2 ip route
wsl-gvproxy.exe exists at /mnt/c/Users/ihe1lo/wsl-vpnkit/wsl-gvproxy.exe
starting gvproxy at /mnt/c/bin/wsl-gvproxy.exe...
started gvproxy
waiting for dhcp...
time="2022-03-17T07:50:24Z" level=info msg="interface eth1 prevented this program to run"

Any Idear whats going on?

**Comment by sakai135** [comment, marked as answer]
What do you get when running `wsl.exe --list --verbose` and `wsl.exe --status` ?

```
  NAME          STATE           VERSION
* Ubuntu        Stopped         2
  wsl-vpnkit    Stopped         2
```

```
Default Distribution: Ubuntu
Default Version: 2
WSL version: 0.56.1.0
Kernel version: 5.10.93.2
WSLg version: 1.0.30
MSRDC version: 1.2.2924
Direct3D version: 1.601.0
Windows version: 10.0.22572.201
```

**Reply by seb4nihel** [reply]
I just realized, that i imported the system as wsl1 instead of wsl2, changed it and it worked as intended.
